bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableHBaseVersion.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.hbase;
+
+import com.google.api.core.InternalApi;
+import com.google.cloud.bigtable.hbase.util.Logger;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/** For internal use only - public for technical reasons. */
+@InternalApi("For internal usage only")
+public class BigtableHBaseVersion {
+
+  private static final Logger LOG = new Logger(BigtableHBaseVersion.class);
+
+  public static final String CLIENT_VERSION = getVersion();
+
+  public static final String USER_AGENT_VALUE = "java-bigtable-hbase-" + getVersion();
```

Comment:
I don't think we should conflate version with user agent. Lets keep the user agent defined in the settings